doc/release-notes/4225-stale-solr-records.md

```diff
@@ -0,0 +1,7 @@
+## Major Use Cases
+
+- Administrators will be able to use new APIs to keep the Solr index and the DB in sync, allowing easier resolution of an issue that would occasionally cause search results to not load. 
+
+## Notes for Dataverse Installation Administrators
+
+New API endpoints... point to Guides
```

Comment:
We should fill this in before it goes to QA. If my suggestion below is followed, we would point to http://guides.dataverse.org/en/latest/admin/solr-search-index.html